Libraries/Database/QueryExtensions.cs

```diff
@@ -39,5 +39,21 @@ public static Task<IEnumerable<ContentItem>> PaginateAsync(this IQuery<ContentIt
         public static Task<IEnumerable<TPart>> PaginateAsync<TPart>(this IQuery<ContentItem> query, int pageIndex = 0, int count = int.MaxValue)
             where TPart : ContentPart =>
             PaginateAsync(query, pageIndex, count).ContinueWith(t => t.Result.As<TPart>().Where(part => part != null), TaskScheduler.Default);
+
+        /// <summary>
+        /// Breaks the query up into pages and lists the page using the given zero-based index.If pageIndex is 0 and
+        /// count is <see cref="int.MaxValue"/> then the whole query is listed.
+        /// </summary>
+        /// <param name="query">The query to paginate.</param>
+        /// <param name="skip">Number of items to skip. Can be null.</param>
+        /// <param name="count">The page size. Can be null.</param>
+        /// <returns>The desired page of the resulting <see cref="ContentItem"/>s converted into the desired ContentPart. Those
+        /// that don't have it are discarded.</returns>
+        public static Task<IEnumerable<ContentItem>> SliceAsync(this IQuery<ContentItem> query, int? skip = 0, int? count = int.MaxValue)
+        {
+            if (skip is not null and > 0) query = query.Skip(skip.Value);

```

Comment:
Azért ezt a syntaxot még nekem is szokni kell.

Reply:
Nem csak neked tetya.